packages/babeld-auto-gw-mode/Makefile

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2021 Santiago Piccinini <spiccinini@altermundi.net>
+#
+# This is free software, licensed under the GNU Affero General Public License v3.
+#
+
+include ../../libremesh.mk
+
+define Package/$(PKG_NAME)
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Bebeld auto Internet gateway module.
+  MAINTAINER:=Santiago Piccinini <spiccinini@altermundi.net>
+  DEPENDS:=+libubus-lua +watchping +ip +lime-proto-babeld
```

Comment:
`+ip` toma el de busybox o el de iproute2 ?

Reply:
you are right, ip uses iproute2, I am removing it.